R/bundle.R

```diff
@@ -360,6 +360,10 @@ isShinyRmd <- function(filename) {
       # ...and "runtime: shiny", then it's a dynamic Rmd.
       return(TRUE)
     }
+    if (any(grepl('^shiny', unlist(yaml)))) {
```

Comment:
I think this is too likely to lead to false positives; there are a lot of things that could start with "shiny" but aren't indentation errors. Maybe just check for the exact string `shiny` here?